provisioning/playbooks/elastic/setup_cb_transport.yml

```diff
@@ -0,0 +1,49 @@
+---
+- hosts: elastic
+  sudo: True
+  vars:
+    cb_transport_plugin_version: 2.0.0
```

Comment:
Same reason as the couchbase create_bucket file, these variables are specific to setting up the Couchbase Transport plugin.